### PR TITLE
Allow extra args to test command

### PR DESCRIPTION
**Pull request by lucasmafra**
This will allow giving extra args when running tests, such as `flutter test --no-sound-null-safety`

### CODE REVIEW

File: lsp-dart.el [hunk 1]
```diff
@@ -60,6 +60,11 @@
   :type '(repeat string)
   :group 'lsp-dart)
 
+(defcustom lsp-dart-test-extra-args '()
+  "Extra arguments to be passed to test command (e.g: --no-sound-null-safety)."
+  :type '(repeat string)
+  :group 'lsp-dart)
+
```
Comment (ericdallo) [inline lsp-dart.el:67]: Since this defcustom is specific for `lsp-dart-test` could you move it to `lsp-dart-test.el`?

Comment (lucasmafra) [reply lsp-dart.el]: Done

**Review by ericdallo — APPROVED** [review]
